apps/user-office-frontend/src/components/proposal/ProposalTableOfficer.tsx

```diff
@@ -110,6 +110,18 @@ let columns: Column<ProposalViewData>[] = [
     field: 'title',
     ...{ width: 'auto' },
   },
+  {
+    render: (principalInvestigator) => {
+      return `${principalInvestigator.principalInvestigator?.lastname}, ${principalInvestigator.principalInvestigator?.firstname}`;
```

Comment:
I think because of the `,` in between this will most likely pass any empty check.

Reply:
Ah yea, that makes sense. Do you know a way to deal with the potential null entries?

Reply:
we can just test like  principalInvestigator.principalInvestigator?.firstname 
 and principalInvestigator.principalInvestigator?.lastname? `${principalInvestigator.principalInvestigator?.lastname}, ${principalInvestigator.principalInvestigator?.firstname}`: empty string;

Reply:
Struggling to follow a bit here Farai. Is this what you meant?
![image](https://user-images.githubusercontent.com/20170535/230059828-3b338d1e-f96e-4cb1-a92f-f0518b218915.png)

Reply:
Yes but more like this will work `     
    render: (principalInvestigator) => {
      if (
        principalInvestigator.principalInvestigator?.lastname &&
        principalInvestigator.principalInvestigator?.firstname
      ) {
        return `${principalInvestigator.principalInvestigator?.lastname}, ${principalInvestigator.principalInvestigator?.firstname}`;
      }

      return '';
    }`

Reply:
Perfect, thanks for clarifying there

Reply:
Thank sorry we still need this  `emptyValue: '-',` in both of them as well if we still want to show `-` when no value is available.